app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -428,6 +415,15 @@ protected String doInBackground(String... params) {
 
                 Log.v("Stage 3", "Pdf writer");
 
+                if (mPassword != null && !mPassword.trim().equals("")) {
+                    writer.setEncryption(mPassword.getBytes(),
+                            getString(R.string.app_name).getBytes(),
```

Comment:
Is there any reason for setting app name as the owner password?

Reply:
No. Can you suggest any other option, which we can set as the owner password? @Swati4star

Reply:
No problem. Let it be the owner password only.